lib/ff_dpdk_if.c

```diff
@@ -1195,8 +1208,12 @@ main_loop(void *arg)
 
             nb_rx = rte_eth_rx_burst(port_id, queue_id, pkts_burst,
                 MAX_PKT_BURST);
-            if (nb_rx == 0)
+            if (nb_rx == 0) {
+                ff_status.norx++;
                 continue;
+            }
+
+            ff_status.dorx += nb_rx;
```

Comment:
It may not be  inaccurate if just counting the number of rx/norx.
It's better to save tsc, such as idle_tsc, work_tsc, but it is more difficult.